yb-voyager/cmd/constants.go

```diff
@@ -20,7 +20,7 @@ const (
 	SPLIT_FILE_CHANNEL_SIZE     = 20
 	META_INFO_DIR_NAME          = "metainfo"
 	NEWLINE                     = '\n'
-	GET_SERVERS_QUERY           = "SELECT * FROM yb_servers()"
+	GET_YB_SERVERS_QUERY        = "SELECT host, port, num_connections, node_type, cloud, region, zone, public_ip FROM yb_servers();"
```

Comment:
Nit: No need of semi-colon in query string. Driver takes care of that part while running the query